### PR TITLE
Structured Readme for ID2

### PR DESCRIPTION
**Pull request by ets994**
ID2 has come to an end.  Here is the structured readme outlining all of the work done.

### CODE REVIEW

File: documents/ID2/ID2Readme.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+[Here](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/tree/develop/src) is the link to the current source code for the project, and [Here](https://dev.braunson.me/) is a link to the current development build of the software.
+
+# Mini-Milestones
+
+[Our current list](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/ID2/ID3%20Mini-Milestones.pdf) of mini-milestones along with the roadmap for their completion.  This document also includes a break down of how our timelogging will work in the future deliverables.  Various categories were merged as they were viewed as redundant.
+
+# Risk Report
+
+[Our list of current risks](TODO GET RISK DOC MERGED) is here.
```
Comment (camilleion) [inline documents/ID2/ID2Readme.md]: https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/ID2/Top%2010%20Risks%20followed%20by%20mitigation%20plan%20for%20ID2.pdf

File: README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 For more information about ID1 and the location of its contents, refer to [here](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/tree/ID1/documents/ID1).
 
+## Incremental Deliverable 2
+
+For more information about ID2 and the location of its contents, refer to [here]().
```
Comment (braunsonm) [inline README.md]: Assuming you named the tag ID2, the link will be:
https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/tree/ID2/documents/ID2

File: documents/ID2/ID2Readme.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+# Risk Report
+
+[Our list of current risks](TODO GET RISK DOC MERGED) is here. 
+
+# Reviews Conducted
+
+[This](hhttps://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/reviews/ID2FormalReview.pdf) is the formal review conducted for ID2. It was done in more depth, and more formally than the review for ID1. [Here](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/meetings/agenda10.15.md) is the meeting in which the agreement was made for the design document to be the artifact that underwent formal review, and [here](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/reviews/DocReview.tex) is the updated review document.  Finally [here](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/pull/56) is the final pull request where the document was checked over one last time to ensure that all required changes were made.
```
Comment (camilleion) [inline documents/ID2/ID2Readme.md]: this url has an extra 'h' in 'https'

File: documents/ID2/ID2Readme.md (hunk 1 above)
Comment (camilleion) [inline documents/ID2/ID2Readme.md]: https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/reviews/ID2FormalReview.pdf

File: documents/ID2/ID2Readme.md (hunk 3 above)
Comment (ets994) [reply documents/ID2/ID2Readme.md]: fixed

File: documents/ID2/ID2Readme.md (hunk 1 above)
Comment (ets994) [reply documents/ID2/ID2Readme.md]: fixed

File: README.md (hunk 2 above)
Comment (ets994) [reply README.md]: done!

**Comment by eileenvanheerde** [comment]
Looking good

**Pull request merged**